flake.nix

```diff
@@ -76,6 +77,15 @@
             MD031 = false;
             MD032 = false;
           };
+          lua-ls = {
+            config = {
+              Lua = {
+                workspace.library = with pkgs; [
+                  lua51Packages.dkjson
```

Comment:
can this be aliased to baseDependency or luarocks-tag-release.buildInputs ? so that it doesn't get out of sync with luarocks-tag-release's buildInputs